site/app/page.tsx

```diff
@@ -83,18 +95,49 @@ export function Dialog(props: {
 ]
 
 const initialTransformSource = `
-import { Project } from 'ts-morph';
-import { getJsxElement, getPropTypes } from '@tsxmod/utils';
+import { JsxElement, JsxSelfClosingElement, Node, Project } from 'ts-morph'
+import { getJsxElement, getJsxElements, getPropTypes } from '@tsxmod/utils'
 
 export default function (project: Project) {
-  const appSourceFile = project.getSourceFileOrThrow('App.tsx');
-  const buttonSourceFile = project.getSourceFileOrThrow('Button.tsx');
-  const dialogSourceFile = project.getSourceFileOrThrow('Dialog.tsx');
-  
+  const appSourceFile = project.getSourceFileOrThrow('App.tsx')
+  const buttonSourceFile = project.getSourceFileOrThrow('Button.tsx')
+  const dialogSourceFile = project.getSourceFileOrThrow('Dialog.tsx')
+
   const appElement = getJsxElement(appSourceFile, 'App')
-  const buttonPropTypes = getPropTypes(buttonSourceFile.getVariableDeclaration('Button'))
-  
+  const buttonPropTypes = getPropTypes(
+    buttonSourceFile.getVariableDeclaration('Button')
+  )
+
   // Write codemod here
+  const deprecatedButtonProps = buttonPropTypes.filter((propType) =>
```

Comment:
actually I think the current code mod is wrong, we should just analyse element prop types in `App.tsx` and migrate based on deprecation message, in this way we can migrate the whole app, but I don't know how to achieve it yet, super new to ts-morph and ts compiler, will check later